lib/database.js

```diff
@@ -1,40 +1,40 @@
 const path = require('path');
 
+const Sequelize = require('sequelize');
 const { Client } = require('pg');
 const Umzug = require('umzug');
 
-const SequelizeDb = require('../models');
-
 const env = process.env.NODE_ENV || 'development';
 const DEFAULT_CONFIG = require('../config/config')[env];
 
 class Database {
     constructor(config) {
         this.config = config || DEFAULT_CONFIG;
-        this.SequelizeDb = SequelizeDb;
+        this.SequelizeDb = Database.createSeqelize(config);
+    }
+
+    static createSeqelize(config = DEFAULT_CONFIG) {
```

Comment:
Not sure if there's a better place for this? I think my goal for this file was to abstract away `Sequelize` db management.

Been a while since I used this and also weird to see there's a distinction between a `Sequelize` vs. `Sequelize` db.